app/controllers/facility_orders_controller.rb

```diff
@@ -147,6 +147,8 @@ def load_cross_core_order_details
         @cross_core_orders_by_facility[order_facility] = order
       end
     end
+
+    @project_name = project&.name
```

Comment:
Is the `&` here to handle single-facility projects, or is there another case?
If it's just that, I would vote for something a little more verbose but intention revealing:
```
  if project.present?
    project_orders = project.orders.where.not(facility_id: @order.facility_id)
    project_orders.each do |order|
      order_facility = order.facility.to_s
      @cross_core_order_details_by_facility[order_facility] = order.order_details.ordered_by_parents
      @cross_core_orders_by_facility[order_facility] = order
    end
    @project_name = project.name
  else
    @project_name = nil
  end
```
also, maybe this should be `@cross_core_project`?